pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/internal/TopicsImpl.java

```diff
@@ -199,20 +204,62 @@ public CompletableFuture<List<String>> getListAsync(String namespace, TopicDomai
                 .queryParam("includeSystemTopic", options.isIncludeSystemTopic());
         final CompletableFuture<List<String>> persistentList;
         final CompletableFuture<List<String>> nonPersistentList;
+        final CompletableFuture<List<String>> nonPersistentPartitionedList;
```

Comment:
why don't we have the same behaviour for "persistent" partitioned topics ?
I would expect here the same code but "persistentPartitionedList"

it looks like we are simulating something on the client.

Reply:
persistentList retrieves all the topics which includes partitioned and non-partitioned "persistent" topics(/persistent/tenant/namespace).
getPersistentList retrives data from the path /managed-ledger-path/tenant/namespace/persistent which contains list of both partitioned and non-partitioned.
But nonPersistentList retrieves topics only from the namespace owned bundle and rest of the bundle topics were skipped.
That is the reason we are making non-persistent/tenant/namespace/partitioned call from the client to include non persistent partitioned topics.

Reply:
Hi @eolivelli ,
Can you please confirm as the current API lists both partitioned and non partitioned topics, so we don't need to add explicitly.
let me know if I need to change any
Thanks